rust/repositories.bzl

```diff
@@ -5,7 +5,7 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 DEFAULT_TOOLCHAIN_NAME_PREFIX = "toolchain_for"
 
-def rust_repositories(version="1.39.0", iso_date=None, rustfmt_version="1.4.8"):
+def rust_repositories(version = "1.39.0", iso_date = None, rustfmt_version = "1.4.8"):
```

Comment:
Is that a change by the starlark formatter?

Reply:
Yes it is.  To confirm, I checked out master and ran buildifier on this file, and this was the resulting change.

Reply:
Thanks!